### DISCUSSION TITLE
Stopping support for Maya 2018 and VS 2015

### DISCUSSION THREAD

**Discussion by kxl-adsk** [Autodesk/maya-usd#1217] [Announcement]
As some of you noticed, we have a large matrix of supported combinations of compiler / USD / Maya versions. This matrix continues to grow making validation and development for all the combinations impossible.
Our Technical Steering Committee has decided to stop support for Maya 2018 and bump the minimal compiler version for Windows to VS 2017.

**Comment by koen-v** [comment]
Hello Autodesk,

Is it known what the last CL is to support maya 2018?

Thanks,
Koen

**Reply by kxl-adsk** [reply]
I’m aware of this issue https://github.com/Autodesk/maya-usd/issues/1215. 

But I like your question - it’s would be better to indicate clearly what is the last commit that supports Maya 2018. I will check it tomorrow and see how much there needs to be fixed. We either bring it back to buildable state or find the commit that works. Then I change the documentation and remove Maya 2018 from the list. 

Stay tuned.

**Comment by koen-v** [comment]
Thanks for looking into this.
if there is any way we could get the 0.8.0 tag to still compile and then we sunset 2018 support for any releases after this, it would work great for us ( lots of little good things in there).

Cheers,
Koen

**Reply by kxl-adsk** [reply]
https://github.com/Autodesk/maya-usd/pull/1225 will bring v0.8.0 back to a state where you can compile it against Maya 2018.

Given that v0.8.0 is already tagged, we can't change the commit it points to. With the above PR, we will have the last commit to refer to, but I will see if we can make v0.9.0 the last version supporting Maya 2018. It would be a cleaner cut.

**Comment by koen-v** [comment]
Fantastic, thank you very much, I think this is a much more elegant way to retire 2018 support, and this will leave our 2018 in a pretty good state the time we will still need it.

Cheers,
Koen

**Reply by kxl-adsk** [reply]
We have tagged the 0.9.0 release. This is the last release supporting Maya 2018.

**Reply by koen-v** [reply]
Perfect, thanks very much
